tests/unit/moduleapi/keyspecs.tcl

```diff
@@ -5,17 +5,17 @@ start_server {tags {"modules"}} {
 
     test "Module key specs: Legacy" {
         set reply [r command info kspec.legacy]
-        assert_equal $reply {{kspec.legacy -1 {} 1 2 1 {} {{flags read begin_search {type index spec {index 1}} find_keys {type range spec {lastkey 0 keystep 1 limit 0}}} {flags write begin_search {type index spec {index 2}} find_keys {type range spec {lastkey 0 keystep 1 limit 0}}}} {}}}
+        assert_equal $reply {{kspec.legacy -1 {} 1 2 1 {} {summary {} complexity {} since {} group module key-specs {{flags read begin-search {type index spec {index 1}} find-keys {type range spec {lastkey 0 keystep 1 limit 0}}} {flags write begin-search {type index spec {index 2}} find-keys {type range spec {lastkey 0 keystep 1 limit 0}}}}}}}
```

Comment:
maybe just `lindex` or `dict get` the relevant entry, instead of matching the whole reply?